c++/src/kj/async-io.c++

```diff
@@ -1816,6 +1835,35 @@ Promise<Own<AsyncCapabilityStream>> AsyncCapabilityStream::receiveStream() {
   });
 }
 
+kj::Promise<Maybe<Own<AsyncCapabilityStream>>> AsyncCapabilityStream::tryReceiveStream() {
+  struct ResultHolder {
+    byte b;
+    Own<AsyncCapabilityStream> stream;
+  };
+  auto result = kj::heap<ResultHolder>();
+  auto promise = tryReadWithStreams(&result->b, 1, 1, &result->stream, 1);
```

Comment:
You don't need to do anything about it, but passing the same un-named, un-commented literal into a function 3 times is making me sad :(

Reply:
I'm tempted to redo the async I/O interfaces to favor `ArrayPtr<byte>` over `void*, size_t`, which would improve things like this. Main thing holding me back is the amount of code that would need to be updated...